src/python/grapl_analyzerlib/grapl_analyzerlib/nodes/network_connection.py

```diff
@@ -51,36 +51,38 @@ def self_type() -> str:
 
 
 class NetworkConnectionQuery(EntityQuery[IPPV, IPPQ]):
-
-    @with_int_prop('port')
+    @with_int_prop("port")
     def with_port(
-            self,
-            *,
-            eq: Optional["IntOrNot"] = None,
-            gt: Optional["IntOrNot"] = None,
-            ge: Optional["IntOrNot"] = None,
-            lt: Optional["IntOrNot"] = None,
-            le: Optional["IntOrNot"] = None,
+        self,
+        *,
+        eq: Optional["IntOrNot"] = None,
+        gt: Optional["IntOrNot"] = None,
+        ge: Optional["IntOrNot"] = None,
+        lt: Optional["IntOrNot"] = None,
+        le: Optional["IntOrNot"] = None,
     ):
         pass
 
-    @with_str_prop('ip_address')
+    @with_str_prop("ip_address")
     def with_ip_address(
-            self,
-            *,
-            eq: Optional["StrOrNot"] = None,
-            contains: Optional["OneOrMany[StrOrNot]"] = None,
-            starts_with: Optional["StrOrNot"] = None,
-            ends_with: Optional["StrOrNot"] = None,
-            regexp: Optional["OneOrMany[StrOrNot]"] = None,
-            distance_lt: Optional[Tuple[str, int]] = None,
+        self,
+        *,
+        eq: Optional["StrOrNot"] = None,
+        contains: Optional["OneOrMany[StrOrNot]"] = None,
+        starts_with: Optional["StrOrNot"] = None,
+        ends_with: Optional["StrOrNot"] = None,
+        regexp: Optional["OneOrMany[StrOrNot]"] = None,
+        distance_lt: Optional[Tuple[str, int]] = None,
```

Comment:
I'm not gonna lie - I don't love OneOrMany.

I'd rather we just stick to `distance_lt=[one_thing]` or something, even if it's a tad verbose. Not a blocker, just an opinion

Reply:
Yeah, might be worth discussing the tradeoffs. I feel like "one" is the majority case, so optimizing the API for that makes sense.